src/modules/Channel/components/Message/index.tsx

```diff
@@ -125,6 +126,8 @@ const Message = ({
     setIsScrolled,
   });
 
+  const { isLoaded } = useLazyImageLoader(messageScrollRef);
```

Comment:
It seems that messages unrelated to images are also being re-rendered by the IntersectionObserver.